Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2574,6 +2577,49 @@ def to_context(self):
 
             return ret_value
 
+    class EMAIL(Indicator):
+        """
+        EMAIL indicator class
```

Comment:
please add the `context-standards` xsoar.pan.dev link.

Reply:
I think this is still missing